test/blockchaintest/blockchaintest.hpp

```diff
@@ -0,0 +1,75 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+#pragma once
+
+#include "../state/bloom_filter.hpp"
+#include "../state/state.hpp"
+#include <evmc/evmc.hpp>
+#include <vector>
+
+#include <nlohmann/json.hpp>
+namespace json = nlohmann;
+
+namespace evmone::test
+{
+
+// https://ethereum.org/en/developers/docs/blocks/
+struct BlockHeader
+{
+    hash256 parent_hash;
+    address coinbase;
+    hash256 state_root;
+    hash256 receipts_root;
+    state::BloomFilter logs_bloom;
+    int64_t difficulty;
+    bytes32 prev_randao;
+    int64_t block_number;
```

Comment:
```suggestion
    int64_t number;
```

Reply:
The members name were taken from official ethereum documentation. I can change them but it worth considering again https://ethereum.org/en/developers/docs/blocks/#block-anatomy

Reply:
ok